src/Components/StepMarker/types.ts

```diff
@@ -0,0 +1,24 @@
+import { ReactNode } from "react"
+
+export interface StepProps {
+  label?: string
+  isActive?: boolean
+  isComplete?: boolean
```

Comment:
Are these all optional or if they are `undefined` it'll just look like a  "⚪️▫️⚪️▫️⚪️"?

Reply:
Heh, no they're all required, need to update this as it was a dev thing